walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -0,0 +1,187 @@
+. Verify that the *Access Type* field is set to `public`. This means the client is a frontend application that needs to login via a web browser.
+
+[type=verification]
+Select the *Clients* item in the side menu. The `order-entry-system` client should be listed and *Enabled* should be `True`.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above and that the *Enabled* toggle in the *Client* is set to *ON*. If you are still having issues, contact your administrator.
+
+=== Create a User
+
+Red Hat Single Sign-On provides identity brokering functionality. Identity
+brokering facilitates login using a shared identity. For example, it's possible
+to create an *Identity Provider* in a *Realm* in SSO to enable login
+using social identities such as Google, Twitter, or GitHub. When the user
+chooses to login using an *Identity Provider* a *User* is automatically created
+in the SSO *Realm* with the details from their chosen
+*Identity Provider*.
+
+In this Solution Pattern a *User* will be created manually to keep things
+simple.
+
+. Navigate to the link:{sso-realm-url}[SSO Realm, window="_blank"].
+. Enter the username `{user-username}` and password `password` if prompted.
+. Select the *Users* entry in the side menu.
+. Click *Add user*.
+.. Enter the username `customer`.
+.. Leave other fields with their default values and click *Save*.
+. The `customer` user details should now be displayed with tabs along the top of the UI.
+. Select the *Credentials* tab.
+.. Enter the password `customer-password` in the *New Password* field.
+.. Enter the same password in the *Password Confirmation* field.
+.. Set the *Temporary* toggle to the *OFF* position.
++
+NOTE: If we left *Temporary* in the *ON* position the user would be forced to create a new password when they perform a login.
+.. Click the *Reset Password* button.
+.. When prompted click the *Change password* button in the modal.
+
+[type=verification]
+Select the *Users* item in the side menu. Click the *View all users* button. The `customer` user should be listed.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+[time=15]
+== Enabling SSO in the Order Entry System
+
+=== Obtaining the SSO Configuration
+
+To secure an application with SSO a *Client Adapter* is required.
+Various platforms are supported with *Client Adapters*:
+
+. Spring Boot
+. Node.js
+. JBoss EAP
+. Fuse
+. JavaScript (client-side)
+. Servlet Filter
+
+{blank}
+
+The *Order Entry System* is run from a Node.js server, so the Node.js
+`keycloak-connect` adapter is included in the code. The following steps will
+demonstrate how to include a configuration and enable the adapter.
++
+NOTE: Enabling the adapter can vary depending on how an application has been coded. The Order Entry System has been developed to enable SSO protection if a configuration is found in the environment on startup.
```

Comment:
This render incorrectly (note the + before the note):

![note](https://user-images.githubusercontent.com/1851198/70332109-29b6ad00-1841-11ea-8035-f982c66cf1c9.png)